locales/templates/app.pot

```diff
@@ -126,3 +136,77 @@ msgctxt "preference_mozilla_telemetry_summary"
 msgid "Learn more"
 msgstr ""
 
+#. Content description (not visible, for screen readers etc.): Clear text in
+#. URL bar
+msgctxt "content_description_clear_input"
+msgid "Clear input"
+msgstr ""
+
+#. Content description (not visible, for screen readers etc.): Clicking on the
+#. website overlay      while in typing mode will dismiss and go back to the
+#. browser. This text will be read when the      overlay if focused. See
+#. https://github.com/mozilla-mobile/focus-android/issues/50
+msgctxt "content_description_dismiss_input"
+msgid "Dismiss"
+msgstr ""
+
+#. Content description (not visible, for screen readers etc.): Erase button:
+#. Erase the browsing      history and go back to the home screen.
+msgctxt "content_description_erase"
+msgid "Erase browsing history"
+msgstr ""
+
+#. Content description (not visible, for screen readers etc.): Lock icon
+#. showed in the URL bar for      https pages..
+msgctxt "content_description_lock"
+msgid "Secure connection"
+msgstr ""
+
+#. This text is not visible. Instead it will be read by the accessibility
+#. service once we start      loading a website
+msgctxt "accessibility_announcement_loading"
+msgid "Loading"
+msgstr ""
+
+#. This text is not visible. Instead it will be read by the accessibility
+#. service once we      finished loading a website
+msgctxt "accessibility_announcement_loading_finished"
+msgid "Website loaded"
+msgstr ""
+
+#. Content description (not visible, for screen readers etc.): "Three dot"
+#. menu button.
+msgctxt "content_description_menu"
+msgid "More options"
+msgstr ""
+
+#. Content description (not visible, for screen readers etc.): Navigate
+#. forward (browsing history)
+msgctxt "content_description_forward"
+msgid "Navigate forward"
+msgstr ""
+
+#. Content description (not visible, for screen readers etc.): Reload current
+#. website
+msgctxt "content_description_reload"
+msgid "Reload website"
+msgstr ""
+
+msgctxt "firstrun_title"
+msgid "Browse like no one’s watching."
+msgstr ""
+
+msgctxt "firstrun_description"
+msgid ""
+"Automatically block online trackers while you browse. Then tap to erase "
+"visited pages, searches, cookies and passwords from your device."
+msgstr ""
+
+msgctxt "firstrun_button"
+msgid "Ok, got it!"
+msgstr ""
+
+msgctxt "firstrun_footer"
+msgid "To be clear, sites and service you use can still know where you've been."
```

Comment:
CC: @antlam (UX), @ahunt (Dev) - Implemented in https://github.com/mozilla-mobile/focus-android/issues/36